cloudformation/ELK_Stack_Multi_AZ_in_Private_VPC.json

```diff
@@ -528,9 +535,12 @@
                             "wget -O /opt/logcabin/config.js https://raw.githubusercontent.com/guardian/elk-stack/master/config/config.js",
                             { "Fn::Join": [ "", [ "sed -i",
                                 " -e s,@@LOGCABIN_HOST,",
-                                    { "Fn::If" : [ "HasDNS",
-                                        { "Fn::Join": [".", [ "kibana", {"Ref": "HostedZoneName"} ]] },
-                                        { "Fn::GetAtt": [ "ElkPublicLoadBalancer", "DNSName" ] }
+                                    { "Fn::Join": ["", 
+                                        [{ "Fn::If" : [ "HasSSLCertificate", "https://", "http://" ] },
+                                        { "Fn::If" : [ "HasDNS",
+                                            { "Fn::Join": [".", [ "kibana", {"Ref": "HostedZoneName"} ]] },
+                                            { "Fn::GetAtt": [ "ElkPublicLoadBalancer", "DNSName" ]}
+                                        ]}]
```

Comment:
I am not a huge fan of the nested `Fn:...` but I am not aware of any other way to do so with cfn template